packages/react-data-grid/src/Row.tsx

```diff
@@ -47,7 +47,11 @@ export default class Row extends React.Component<RowRendererProps> implements Ro
 
     const cellProps: CellRendererProps & { ref: (cell: CellRenderer | null) => void } = {
       ref: (cell) => {
```

Comment:
Do we have any preference? Your code vs this
```
(cell) => cell ? this.cells.set(key, cell) : this.cells.delete(key);
```

Reply:
Both are good IMO.